static/index.html

```diff
@@ -1295,6 +1295,25 @@ <h3 class="mb-0">2 x 1 Box</h3>
       </div>
     </section>
 
+    <div class="cc">
+      <div class="cc">
+        <div class="cc bg-poppy-light rs-mb-2">
+          <div class="rs-p-6">
+            Yvonne's approach for nested cc
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="box">
+      <div class="box">
+        <div class="box bg-sky-light">
+          <div class="rs-p-6">
+            Mike's approach for nested cc
```

Comment:
```suggestion
            Mike's approach for nested containers
```

Reply:
Oh this is for internal conversation and preview only - I fully intend to clean this up @sherakama 😂